src/zenml/integrations/mlflow/services/mlflow_deployment.py

```diff
@@ -67,12 +68,16 @@ class MLFlowDeploymentConfig(LocalDaemonServiceConfig):
         mlserver: set to True to use the MLflow MLServer backend (see
             https://github.com/SeldonIO/MLServer). If False, the
             MLflow built-in scoring server will be used.
+        timeout: the number of seconds to wait for the service to start/stop.
+        deploy_decision: whether to deploy the model or not
     """
 
     model_uri: str
     model_name: str
     workers: int = 1
     mlserver: bool = False
+    timeout: int = 10
+    deploy_decision: bool
```

Comment:
This should be left out of the service. It's only relevant in the pipeline step dynamics, not as a service configuration.